src/main/java/com/devonfw/ide/sonarqube/common/impl/DevonPlugin.java

```diff
@@ -1,17 +0,0 @@
-package com.devonfw.ide.sonarqube.common.impl;
-
-import org.sonar.api.Plugin;
-
-/***
- * {@link Plugin} that registeres the Devon rules in SonarQube.
- */
-public class DevonPlugin implements Plugin {
-
-  @Override
-  public void define(Context context) {
-
-    context.addExtension(DevonSonarDefinition.class);
-    context.addExtension(DevonSonarRegistrar.class);
```

Comment:
IMHO we need to put these two lines into the other plugin class so we do not loose this functionality.

Reply:
IMHO we should also rename the `DevonSonarPlugin` class to `SonarDevon4jPlugin`. Is that class referenced in our POM? Then we would need to update there as well.

Reply:
The functionality won't be lost as those two classes are already added as an extension in the DevonSonarPlugin class. The renaming of the class is a good idea though, I will fix that asap.